web/components/input/buttons/CheckboxGroup/Checkbox.sass

```diff
@@ -1,54 +1,49 @@
+.checkbox-group
+  @extend .flex, .wrap
+
 .checkbox
-  margin-left: 20px;
-  font-family: Apercu Pro;
-  font-size: 18px;
-  font-weight: bold;
-  line-height: 23px;
-  color: $primary;
-  position: relative;
-  margin-bottom: 8px;
-  cursor: pointer;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
+  @extend .user-select-none
+  margin-left: 20px
+  position: relative
+  margin-bottom: $spacing-s
 
   &:hover input:enabled ~ .checkmark
-    background-image: url('../assets/checkbox/hover.svg');
+    background-image: url('../assets/checkbox/hover.svg')
 
   input[type="checkbox"]
-    position: absolute;
-    opacity: 0;
-    cursor: pointer;
+    position: absolute
+    opacity: 0
+    cursor: pointer
 
     &:checked ~ .checkmark
-      background-image: url('../assets/checkbox/checked.svg');
+      background-image: url('../assets/checkbox/checked.svg')
       &:after
-        display: block;
+        display: block
 
     &:disabled
-      cursor: not-allowed;
-      opacity: 0.4;
+      cursor: not-allowed
       &~ .checkmark
-        cursor: not-allowed;
-        opacity: 0.4;
+        cursor: not-allowed
+        opacity: 0.4
         &~ .label-text
-          cursor: not-allowed;
-          opacity: 0.4;
+          cursor: not-allowed
+          opacity: 0.4
```

Comment:
Class, we don't need repeated styles, and if we are using `cursor: not-allowed` etc they should be part of a `unclickable` class that all our buttons can use.